src/java/picard/analysis/artifacts/CollectSequencingArtifactMetrics.java

```diff
@@ -53,12 +53,68 @@
  *
  */
 @CommandLineProgramProperties(
-        usage = CollectSequencingArtifactMetrics.USAGE,
-        usageShort = CollectSequencingArtifactMetrics.USAGE,
+        usage = "This tool examines two sources of sequencing errors, bait-bias and pre-adapter artifacts resulting " +
+                "from hybrid selection protocols.  Hybrid selection enables selection of specific sequences from a pool" +
```

Comment:
"from hybrid selection protocols" -> "from hybrid selection protocols, which are typically used for exome sequencing" 

(if that's correct)